packages/v4/patternfly-docs/content/design-guidelines/components/form/forms.md

```diff
@@ -168,12 +170,12 @@ If all form fields are required, don’t use an asterisk for each one. Instead,
 <img src="./img/required-optional-forms.png" alt="Example of two forms with an indication at the top of whether all fields are required or all fields are optional"/>
 
 
-### User help
+#### User help
 
 There are 3 different methods for providing assistance within a form: [Placeholder text](#placeholder-text), [helper text](#helper-text), and [popovers](#popovers).
 
 
-#### Placeholder text
+##### Placeholder text
 
 <img src="./img/placeholder-text.png" alt="Example of a date placeholder text inside text input field" width="323"/>
 
```

Comment:
Bold the words "Use placeholder text when"
and then maybe make the "never use placeholder" sentence a heading and bullet situation to match

Reply:
@wolfeallison Missed this comment I think!